### PR TITLE
Unforks course about facebook link html removal

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd & @kluo

### CODE REVIEW

File: lms/templates/courseware/course_about_sidebar_header.html [hunk 1]
```diff
@@ -47,6 +47,9 @@
       <a href="${tweet_action}" class="share">
         <span class="icon fa fa-twitter" aria-hidden="true"></span><span class="sr">${_("Tweet that you've enrolled in this course")}</span>
       </a>
+      <a href="${facebook_link}" class="share">
```
Comment (kluo) [inline lms/templates/courseware/course_about_sidebar_header.html]: This context var `facebook_link` was already being passed?

Comment (caesar2164) [reply lms/templates/courseware/course_about_sidebar_header.html]: look further up in this file...

**Comment by stvstnfrd** [comment]
@caesar2164 With this PR, are we completely unforked in this file?

**Comment by stvstnfrd** [comment]
Unless I'm missing something, this isn't just an "unfork"; this is altering our UI.
We weren't previously showing the Facebook link, but now we will?

File: lms/templates/courseware/course_about_sidebar_header.html (hunk 1 above)
Comment (stvstnfrd) [reply lms/templates/courseware/course_about_sidebar_header.html]: @caesar2164 I think the larger question is, are we actually setting `PLATFORM_FACEBOOK_ACCOUNT`?
I didn't know we had a platform Facebook account...

**Comment by caesar2164** [comment]
Closing this as it needs to be an upstream PR first...

**Pull request closed without merging**